packages/notices/src/store/selectors.js

```diff
@@ -29,6 +29,8 @@ const DEFAULT_NOTICES = [];
  *                                      removal without notice.
  * @property {boolean} isDismissible    Whether the notice can be dismissed by
  *                                      user. Defaults to `true`.
+ * @property {string}  type             Type of notice, one of `default`,
+ *                                      or `snackbar`. Defaults to `default`.
  * @property {WPNoticeAction[]} actions User actions to present with notice.
```

Comment:
Just noticed that `speak` is missing as well. Can you please add as part of this PR?